elks/kernel/sys.c

```diff
@@ -24,6 +24,8 @@
 
 static int C_A_D = 1;
 
+void bioshd_park_all(void);
```

Comment:
No forward declarations are allowed in .c files, all would need to go somewhere in the include/linuxmt or include/arch header files. In this case, the function will likely live as a static function within the BIOSHD driver.

The reason for the build failing is that the code below is calling a specific block driver function which isn't part of the configured build. We'll be better off passing a flag from a user mode application that can find its way to the standard block device interface functions, which will eliminate this problem.